deb/openmediavault/workbench/src/app/pages/diagnostics/processes/processes-text-page.component.html

```diff
@@ -0,0 +1,220 @@
+<!-- processes-page.component.html -->
```

Comment:
This file has nothing in common with updating the transloco library. Please remove.